### PR TITLE
feat(md): improve styles on mobile

### PR DESCRIPTION
**Pull request by ranihorev**
Per @luispollo's request, some improvements for the mobile view + fixes for safari (arrrgggh)

### CODE REVIEW

**Comment by luispollo** [comment]
Unfortunately I can't review the CSS changes, but thanks so much for doing this!

File: app/scripts/modules/core/src/managed/Environments2.less [hunk 1]
```diff
@@ -30,4 +35,8 @@
   border: 1px solid var(--color-alto);
   bottom: 4px;
   right: 30px;
+
+  @media (max-width: 768px) {
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/managed/Environments2.less:39]: I'm wondering if we have these breakpoints as css variables?

**Review by christopherthielen — APPROVED** [review]
i'm no css master, but lgtm

**Comment by ranihorev** [comment]
I'll check 👍

**Comment by ranihorev** [comment]
We don't have these breakpoints

**Pull request merged**